src/main/java/com/teamcautionrobotics/robot/ExpanderHatch.java

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
This new line probably does not belong here.
Also, surprise, a wild file appears.
```suggestion
```